setup/symfony_server.rst

```diff
@@ -106,6 +106,15 @@ trust store, registers it in Firefox (this is required only for that browser)
 and creates a default certificate for ``localhost`` and ``127.0.0.1``. In other
 words, it does everything for you.
 
+.. tip::
+
+    If you are doing this in WSL (Windows Subsystem for Linux), the newly created
+    local certificate authority needs to be manually imported in Windows. The file
+    is located in ``wsl`` at ``~/.symfony5/certs/default.p12``. The easiest way to
+    do so is to open ``\\wsl$\Ubuntu\home\`` in ``Explorer`` and then navigate
+    through your username to ``.symfony/certs`` and double-click the ``default.p12``
+    file.
```

Comment:
I recently learned that you can run `explorer.exe` inside the WSL terminal and it'll open the Windows file explorer in the current directory of the shell. Not sure if this is WSL 2 specific (and if anyone is till on WSL 1), but it might be worth using this instead as it's easier.

Reply:
You could even more automate it with executing as a "shortcut" from WSL Bash. It will open the explorer with the correct windows path to the symfony cli certs folder. 

```
explorer.exe `wslpath -w $HOME/.symfony5/certs`
```